Update status matrix for polaris

TheMorc opened the pull request:
2nd try lol
No chinese translation in notes for GPU and USB.

BigfootACA reviewed (changes requested):
if the string is not filled in correctly, an error may be thrown

"GPU and USB doesn't work simultaneously right now so GPU currently works only on the polaris-gpu boot image."
in chinese:
"GPU和USB目前不能同时工作，GPU目前仅适用于polaris-gpu。"

strongtz commented:
> if the string is not filled in correctly, an error may be thrown
> 
> "GPU and USB doesn't work simultaneously right now so GPU currently works only on the polaris-gpu boot image."
> in chinese:
> "GPU和USB目前不能同时工作，GPU目前仅适用于polaris-gpu。"

Please change the state of Bluetooth to `p` as well

TheMorc commented:
> > if the string is not filled in correctly, an error may be thrown
> > "GPU and USB doesn't work simultaneously right now so GPU currently works only on the polaris-gpu boot image."
> > in chinese:
> > "GPU和USB目前不能同时工作，GPU目前仅适用于polaris-gpu。"
> 
> Please change the state of Bluetooth to `p` as well

reverted bluetooth back(firstly i thought that @tjstyle fixed it) and added the translation
